PluralKit.Bot/Commands/Groups.cs

```diff
@@ -353,9 +356,12 @@ public async Task AddRemoveMembers(Context ctx, PKGroup target, AddRemoveOperati
                 await _repo.RemoveMembersFromGroup(conn, target.Id, membersInGroup);
                 
                 if (membersInGroup.Count == members.Count)
-                    await ctx.Reply($"{Emojis.Success} {"members".ToQuantity(membersInGroup.Count)} removed from group.");
-                else 
-                    await ctx.Reply($"{Emojis.Success} {"members".ToQuantity(membersInGroup.Count)} removed from group ({"members".ToQuantity(members.Count - membersInGroup.Count)} already not in group).");
+                    await ctx.Reply(members.Count == 0 ? $"{Emojis.Success} Member removed from group." : $"{Emojis.Success} {"members".ToQuantity(membersInGroup.Count)} removed from group.");
+                else
+                    if (membersInGroup.Count == 0)
+                        await ctx.Reply(members.Count == 1 ? $"{Emojis.Error} Member not removed from group (member already not in group)." : $"{Emojis.Error} No members removed from group (members already not in group).");
```

Comment:
god I wanna move all this text logic out somewhere else this is a disaster lol :) but it works for now